base_layer/core/src/transactions/key_manager/inner.rs

```diff
@@ -1154,25 +1164,104 @@ where TBackend: KeyManagerBackend<PublicKey> + 'static
     pub async fn sign_script_message(
         &self,
         private_key_id: &TariKeyId,
-        challenge: &[u8],
+        nonce: &[u8],
```

Comment:
```suggestion
        challange: &[u8],
```
This is wrong, challenge is correct